ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -398,13 +409,17 @@
               <v-list-item
                 v-if="isRoleStaffReg && isExemptionEnabled"
                 @click="openExemption(TableActions.OPEN_NON_RES_EXEMPTION, item)"
+                data-test-id="non-res-exemption-btn"
               >
                 <v-list-item-subtitle>
                   <img alt="exemption-icon" class="exemption-icon" src="@/assets/svgs/ic_exemption.svg" />
                   <span class="ml-1">Non-Residential Exemption</span>
                 </v-list-item-subtitle>
               </v-list-item>
-              <v-list-item @click="handleAction(item, TableActions.REMOVE)">
+              <v-list-item
+                @click="handleAction(item, TableActions.REMOVE)"
+                data-test-id="remove-mhr-row-btn"
+              >
```

Comment:
We will have some conflicts here, looks like several changes in this area overlap.

Reply:
I can resolve, as your PR is bigger and should go first.